### PR TITLE
Update keyfigures changetext

### PR DESCRIPTION
**Pull request by ssb-cgn**

### CODE REVIEW

File: src/components/KeyFigures/index.jsx [hunk 1]
```diff
@@ -32,6 +43,11 @@ KeyFigures.defaultProps = {
 };
 
 KeyFigures.propTypes = {
+	changes: PropTypes.shape({
+		changeDirection: PropTypes.oneOf(['up', 'down', 'same']),
+		changeText: PropTypes.string,
+		changePeriode: PropTypes.string,
```
Comment (kristofferlund) [inline src/components/KeyFigures/index.jsx]: Should be "changePeriod". "Periode" is Norwegian

File: src/components/KeyFigures/README.md [hunk 2]
```diff
@@ -56,6 +56,24 @@ KeyFigures
...
+        <span class="kf-title">Title</span>
+        <div class="kf-time">2018</div>
+        <div class="number-section">
+            <div class="ssb-number large">789 398</div>
+            <span class="kf-title subtitle">husholdninger</span>
+        </div>
+        <div class="kf-changes">
+             <i class="changes-icon">{feather.[ chevronDown | chevronUp | Minus] 20px}</i>
+            <span class="changes-text">Opp 30 999 kroner</span>&nbsp;
+            <span class="changes-periode">fra året før</span>
```
Comment (kristofferlund) [inline src/components/KeyFigures/README.md]: Should be "changePeriod". "Periode" is Norwegian

File: src/components/KeyFigures/index.jsx (hunk 1 above)
Comment (ssb-cgn) [reply src/components/KeyFigures/index.jsx]: fixed

File: src/components/KeyFigures/README.md (hunk 2 above)
Comment (ssb-cgn) [reply src/components/KeyFigures/README.md]: fixed

**Pull request merged**